.github/workflows/ci_test-conda.yml

```diff
@@ -18,8 +18,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8"]  # previous to last Python version as that one is already used in test-full
```

Comment:
Can we keep this?

As the comment indicates, since "test-full" uses Python 3.9, we use set Python 3.8 for the extra coverage.

Reply:
done